src/Header/AcceptHeaderItem.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+/**
+ * Spiral Framework.
+ *
+ * @license   MIT
+ * @author    Pavel Z
+ */
+
+declare(strict_types=1);
+
+namespace Spiral\Http\Header;
+
+/**
+ * Represents "Accept" header item.
+ */
+class AcceptHeaderItem
+{
+    /** @var string */
+    private $mime = null;
+
+    /** @var string */
+    private $type = null;
+
+    /** @var string */
+    private $subtype = null;
+
+    /** @var float */
+    private $quality = 1.0;
+
+    /** @var array */
+    private $params = [];
+
+    /**
+     * Parse accept header string.
+     *
+     * @param string $string
+     * @return static
+     */
+    public static function fromString(string $string): self
+    {
+        $elements = explode(';', $string);
+
+        $item = new static();
+        $item->setMime(trim(array_shift($elements)));
+
+        foreach ($elements as $element) {
+            [$name, $value] = explode('=', trim($element));
+            $name = trim($name);
+            $value = trim($value);
+
+            if ($name === 'q') {
+                $item->quality = floatval($value);
+            } else {
+                $item->params[$name] = $value;
+            }
+        }
+
+        return $item;
+    }
+
+    /**
+     * @return string
+     */
+    public function __toString(): string
+    {
+        $parts = [$this->getMime()];
+
+        if ($this->quality < 1) {
+            $parts[] = "q=$this->quality";
+        }
+
+        foreach ($this->getParams() as $name => $value) {
+            $parts[] = "$name=$value";
+        }
+
+        return implode('; ', $parts);
+    }
+
+    /**
+     * @param string $mime
+     * @return self
+     */
+    public function setMime(string $mime): self
```

Comment:
Do we really need this setter (to be public)?

Reply:
The setter can be useful if we want to generate "Accept" header by our self, to make some request. It has `__toString()` method for that.